docs/metrix.md

```diff
@@ -239,8 +239,10 @@ The syntax to define a managed generator is:
 generator(id) { // id of the generator which will be managed by Metrix 
   adequacyDownCosts 'ts_cost_down' // Cost of ramping down for the adequacy phase (here a time series name is used) 
   adequacyUpCosts 'ts_cost_up' // Cost of ramping up for the adequacy phase (here a time series name is used) 
-  redispatchingDownCosts 10 // Cost of ramping down (preventive) for the OPF simulation (here a fixed value is used)
-  redispatchingUpCosts 100 // Cost of ramping up (preventive) for the OPF simulation (here a fixed value is used)
+  redispatchingDownCosts 10 // Cost of ramping down for the OPF simulation (here a fixed value is used)
+  redispatchingUpCosts 100 // Cost of ramping up for the OPF simulation (here a fixed value is used)
+  redispatchingDownDoctrineCosts 10 // Doctrine cost of ramping down for the OPF simulation (fixed value or time series name)
+  redispatchingUpDoctrineCosts 100 // Doctrine cost of ramping up for the OPF simulation (fixed value or time series name)
```

Comment:
It might be interesting to add a comment about the difference between these 2 parameters and the 2 existing ones